src/ducks/SANCharts/ChartSettings.js

```diff
@@ -44,18 +46,19 @@ const ChartSettings = ({
           defaultSelected={defaultTimerange}
           className={styles.ranges}
         />
-        <IntervalSelector
-          from={from}
-          to={to}
-          interval={interval}
-          onIntervalChange={onIntervalChange}
-        />
         <CalendarBtn
           onChange={onCalendarChange}
           value={[new Date(from), new Date(to)]}
           className={styles.calendar}
           maxDate={MAX_DATE}
         />
+        <IntervalSelector
+          from={from}
+          to={to}
+          interval={interval}
+          onIntervalChange={onIntervalChange}
+        />
+        <ChartWatchlistContextMenu project={project} />
```

Comment:
@DmitriVanGuard it looks like a bad name for this component. Because you used Chart, you can skip it. Also If you see this name first time you don't understand what does it do.